gallery/src/data-visualization/ReactVis.scss

```diff
@@ -0,0 +1,33 @@
+@import "~react-vis/dist/style";
+
+.nx-donut-hint {
```

Comment:
Are these the standard styles for how we want these to appear? If so this class should be in the lib, not the gallery